test/relations.jl

```diff
@@ -1614,5 +1617,49 @@ end
     @test all(phase_type .== (nt.phase_type for nt in profiles))
 end
 
+@testset "Thermodynamics - optional args" begin
+    ArrayType = Array{Float64}
+    FT = eltype(ArrayType)
+    param_set = parameter_set(FT)
+    profiles = TestedProfiles.PhaseEquilProfiles(param_set, ArrayType)
+    @unpack ρ, e_int, q_tot, T, phase_type = profiles
+    ts =
+        PhaseEquil_ρeq.(
+            param_set,
+            ρ,
+            e_int,
+            q_tot,
+            8,
+            FT(1e-1),
+            RS.NewtonsMethod,
+            T .+ (FT(0.1) .- rand(FT, length(T))),
```

Comment:
D you want this to be unbiased w.r.t. to the true solution, i.e.,
```suggestion
            T .+ (FT(0.1) .* rand(FT, length(T))),
```

Reply:
Ah, yes. This was a last minute typo, thanks!